source/browseMode.py

```diff
@@ -440,9 +442,53 @@ def _iterNodesByType(self,itemType,direction="next",pos=None):
 	def _iterNotLinkBlock(self, direction="next", pos=None):
 		raise NotImplementedError
 
+	def _iterSimilarParagraph(
+			self,
+			kind: str,
+			paragraphFunction: Callable[[textInfos.TextInfo], Optional[Any]],
+			desiredValue: Optional[Any] = None,
+			direction: str = "next",
+			pos: textInfos.TextInfo = None,

```

Comment:
`None` is not a valid value for this parameter, and having a incorrect default here means someone may call this method without specifying `pos`. Can params be reordered here so that specifying incorrect default is not necessary?

Reply:
The way I am using this function is called partial application - this term is from functional programming, not sure if it is used in Pythonic context, but basically in `_quickNavScript` I only resolve the first two function arguments and return the function of the remaining arguements. Even though the order is flexible, it is considered to be a good style to declare function arguments in the order they will be resolved. Moreover pos seems to be the last argument in other similar functions. So But I can remove default values from all arguments.